events/wc_events/wc_test_events.txt

```diff
@@ -0,0 +1,98 @@
+﻿namespace = WCTEST
+
+#Discovers start-up techs
+WCTEST.1 = {
+	type = empty
+	hidden = yes
+
+	immediate = {
+		every_independent_ruler = {
+			limit = {
+				OR = {
+					###IF CULTURES HAVE MAA'S
+					culture = {
+						has_giant_maa_trigger = yes
+						NOT = { has_innovation = innovation_giants }
+					}
+					culture = {
+						has_massive_aerial_maa_trigger = yes
+						NOT = { has_innovation = innovation_flying_beasts }
+					}
+					culture = {
+						has_light_aerial_maa_trigger = yes
+						NOT = { has_innovation = innovation_taming_the_skies }
+					}
+					###RACE SIZE TECHS
+					culture = {
+						is_giant_culture_trigger = yes
+						NOT = { has_innovation = innovation_massive_soldiers }
+					}
+					culture = {
+						is_large_culture_trigger = yes
+						NOT = { has_innovation = innovation_large_soldiers }
+					}
+					culture = {
+						is_small_culture_trigger = yes
+						NOT = { has_innovation = innovation_small_soldiers }
+					}
```

Comment:
If we're really to use it, it just can't work right.
I invited a new orcish clan with a new culture, no tech.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/108757880-dc1c8200-7563-11eb-8826-25e1b97e9558.png)

</details>

Reply:
What do you mean you invited a new orcish clan?  I can add this tech manually to all cultures that would have this tech in history files, but if you make a new culture that is not defined in history it still won't work.

Reply:
> What do you mean you invited a new orcish clan? I can add this tech manually to all cultures that would have this tech in history files, but if you make a new culture that is not defined in history it still won't work.

The Dragonmaw culture, exists in the files.